### ISSUE TITLE
The speed of training is slow?

### ISSUE THREAD

**Issue by HITerStudy** [lukemelas/EfficientNet-PyTorch#10]
Sorry, I found a problem, that was, when I used the pretrained model 'efficientNet-B0' to fine-tune on my own dataset by gpu device, I found the speed of training was slow, thus, is there something wrong? Please give me an answer if having some solutions? Thanks a lot!

**Comment by BowieHsu** [comment]
maybe it's because the group convolution in pytorch is slow? i am new to pytorch, but i found when i train the mobilenetv3 using pytorch, it's slower than resnet50 and resnet34.

**Comment by lukemelas** [comment]
I've posted this on a couple of other issues, but there's an open PyTorch issue that may be relevant. It notes that grouped convs with FP32 are slow (and will be faster soon): pytorch/pytorch#18631

**Comment by ChenXiao61** [comment]
@HITerStudy , Hello, I also want to fine-tune my own limited data using pretrained model 'efficientNet-B0', would you mind telling me how did you do that? 
Best regards,
Xiao Chen

**Comment by HITerStudy** [comment]
> @HITerStudy , Hello, I also want to fine-tune my own limited data using pretrained model 'efficientNet-B0', would you mind telling me how did you do that?
> Best regards,
> Xiao Chen

Of course, I also want to discuss with friends! And, I modified some layers of pretrained model  'efficientNet-B0' to adapt the needs of my own datasets, and other unchanged layers can be trained with the pretrained parameters. Thus, the model can be employed to fine-tune on other datasets.

**Comment by ChenXiao61** [comment]
Well，how many layers did you modify and how about the results of fune-tuning the model?
  


发自我的iPhone

------------------ Original ------------------

**Comment by HITerStudy** [comment]
Two layers, those are, first layer for conv as the channel of image is different, and the last layer for softmax. The performance was ordinary, and the results could be better if some optimization strategies are applied, maybe!


 
---Original---
